database/infrastructure/config.py

```diff
@@ -58,7 +58,7 @@ class vedaDBSettings(BaseSettings):
         description="Boolean if the RDS should be accessed through a proxy",
     )
     rds_type: str = Field(
-        "t3.small",
+        "db.t3.small",
```

Comment:
it expects `t3.small` not `db.t3.small`

Reply:
this came from the base, happy to change it, the edit isn't from me! Maybe I should rebase a new branch--the only changes I made were to the database handler